src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MaintainSupplierPricingControllerTest.kt

```diff
@@ -148,7 +148,28 @@ class MaintainSupplierPricingControllerTest(@Autowired private val wac: WebAppli
       param("moveId", "$fromAgencyId-$toAgencyId")
       param("price", "1O.00")
     }
-      .andExpect { model { attributeHasFieldErrorCode("form", "price", "Invalid price") } }
+      .andExpect { model { attributeHasFieldErrorCode("form", "price", "Pattern") } }
+      .andExpect { model { attribute("contractualYearStart", effectiveYearForDate(effectiveDate).toString()) } }
+      .andExpect { model { attribute("contractualYearEnd", (effectiveYearForDate(effectiveDate) + 1).toString()) } }
+      .andExpect { view { name("add-price") } }
+      .andExpect { status { isOk() } }
+
+    verify(service, never()).addPriceForSupplier(any(), any(), any(), any(), any())
+  }
+
+  @Test
+  internal fun `cannot add price greater than but less than one for Serco`() {
```

Comment:
greater than zero*?

Reply:
Indeed.  Not sure why I didn't do that :facepalm: